AUTHORS

```diff
@@ -2316,5 +2316,378 @@ Ben Newman <ben@meteor.com>
+Juan José Arboleda <soyjuanarbol@gmail.com>
+prodroy1 <prodroy1@in.ibm.com>
+Yusuke Kawasaki <u-suke@kawa.net>
+yuriettys <yres.101@gmail.com>
+sadness_ojisan <sadness.ojisan@gmail.com>
+ka2jun8 <narchbald103@gmail.com>
+sota1235 <sota1235@gmail.com>
+dominikeinkemmer <dominik_einkemmer@gmx.at>
+mki-skt <sakito21@gmail.com>
+Laszlo.Moczo <laszlo.moczo@hu.ibm.com>
+jamesgeorge007 <jamesgeorge998001@gmail.com>
+koki-oshima <koki.oshima@weblio.co.jp>
+takato <taishi.k0903@gmail.com>
+yosuke ota <otameshiyo23@gmail.com>
+tottokotkd <tottokotkd@me.com>
+kazuya kawaguchi <kawakazu80@gmail.com>
+Naojirou Hisada <naojirouhisada@gmail.com>
+tpanthera <bisht11390@gmail.com>
+feng jianmei <fengjianmei@fengs-MacBook-Pro.local>
+DoiChris <pej.deutinger@gmail.com>
+VeysonD <veysond@mercari.com>
+susantruong <susantruong2412@gmail.com>
+Takahiro Nakamura <dxbxdxbxdxb+dev@gmail.com>
+Daiki Arai <darai@yahoo-corp.jp>
+horihiro <hori.kawa.shima.tool+github@gmail.com>
+grimrose <grimrose@users.noreply.github.com>
+timothy searcy <timmyzsearcy@gmail.com>
+nakashima <nakashima1711@gmail.com>
+/Jesse <jesse.katsumata@gmail.com>
+Tadhg Creedon <tadhgcreedon@gmail.com>
+exoego <mogami@exoego.net>
+sigwyg <sigwyg>
+pastak <pasta0915@gmail.com>
+lakatostamas <mail.tamaslakatos@gmail.com>
+Mike Samuel <mikesamuel@gmail.com>
+Suguru Motegi <suguru.motegi@gmail.com>
+Gerhard Stoebich <18708370+Flarna@users.noreply.github.com>
+jy95 <jacques.yakoub@gmail.com>
+Kenigbolo Meya Stephen <expensivestephen@hotmail.com>
+Charles Samborski <demurgos@demurgos.net>
+zhmushan <zhmushan@qq.com>
+yoshimoto koki <yoyoyo427435@gmail.com>
+Ilarion Halushka <hilarion.galushka@gmail.com>
+Yael Hermon <yaelhe@wix.com>
+Mitch Hankins <mitchhankins92@gmail.com>
+Mikko Rantanen <jubjub@jubjubnest.net>
+wenjun ye <1728914873@qq.com>
+briete <briete5814@gmail.com>
+Matteo Collina <hello@matteocollina.com>
+Philipp Renoth <renoth@wildoak.de>
+Audrey Eschright <audrey@npmjs.com>
+Kazushi Kitaya <kazushikitaya@gmail.com>
+peterwmwong <peter.wm.wong@gmail.com>
+Thomas Chung <thomas@nomology.id.au>
+Jordan Harband <ljharb@gmail.com>
+P. Mike <mihan@panfilov.biz>
+Corey Farrell <git@cfware.com>
+Furqan Shaikh <furqan_shaikh_1999@hotmail.com>
+Roy Sommer <roy@sommer.co.il>
+James Bunton <jbunton@atlassian.com>
+Kevin Smith <Kevin.Smith@microsoft.com>
+Minwoo Jung <minwoo@nodesource.com>
+Benjamin Coe <bencoe@gmail.com>
+H1Gdev <h1g.z.ouroboros+github@gmail.com>
+Bidisha Pyne <bidipyne@in.ibm.com>
+Julian Alimin <dmastag@yahoo.com>
+Lakshmi Swetha Gopireddy <lakshmigopireddy@in.ibm.com>
+Manusaporn Treerungroj <m.treerungroj@gmail.com>
+Richard Chamberlain <richard_chamberlain@uk.ibm.com>
+Daniel George Holz <dgholz@gmail.com>
+yathamravali <yatraval@in.ibm.com>
+Tim De Pauw <timdp@users.noreply.github.com>
+Filip Skokan <panva.ip@gmail.com>
+Samuel D. Leslie <sdl@nexiom.net>
+Eric Whitebloom <ewhitebloom@gmail.com>
+Lee Byron <lee@leebyron.com>
+cclauss <cclauss@me.com>
```

Comment:
Same as `cclauss@bluewin.ch` above. @cclauss, which one do you prefer?

Reply:
__Christian Clauss <cclauss@me.com>__ please.  I have dropped the other address.